waiter/src/waiter/scheduler/kubernetes.clj

```diff
@@ -972,6 +1013,32 @@
   (get-services [this]
     (get-services this))
 
+  (signal-instance [this {:keys [id service-id] :as instance} signal-type]
+    (ss/try+
+      (let [service (service-id->service this service-id)
+            {:keys [pod-name]} (unpack-instance-id id)
+            pod (get-in @watch-state [:service-id->pod-id->pod service-id pod-name])
+            service-instance (pod->ServiceInstance this pod)
+            response (signal-service-instance this service-instance service signal-type)]
+        (log/info "qwer" response)
+        ;; response is a pod so it doesnt have :status
+        (if response 
+          (do
+            (scheduler/log-service-instance instance :kill :info)
+            {:success true
+             :message (str signal-type "successfully sent to" id)
+             :status http-200-ok})
+          (do 
```

Comment:
For this branch to execute, `response` is false or nil. There is no `:status` entry in it. The error message should be a more generic `error in sending signal to pod`.

Reply:
I believe if there is a response, the success block will run. If there is no response, then the "non 200 status code" block will run.